cmd/agent/main.go

```
@@ -1,17 +1,30 @@
 package main
 
 import (
+	"flag"
+	"fmt"
 	"time"
 
 	"github.com/as-tanais/observy/internal/agent"
 	models "github.com/as-tanais/observy/internal/model"
 )
 
 func main() {
-	pollInterval := 2 * time.Second
-	reportInterval := 10 * time.Second
 
-	pollsPerReport := int(reportInterval / pollInterval)
+	serverAddr := flag.String("a", "localhost:8080", "Server address host:port, default: localhost:8080")
```

Добавь валидацию значений флагов: интервалы должны быть положительными числами, fail fast при неверных входных данных